x-pack/plugins/lens/public/indexpattern_datasource/field_item.tsx

```diff
@@ -460,7 +460,7 @@ function FieldItemPopoverContents(props: State & FieldItemProps) {
 
   if (props.isLoading) {
     return <EuiLoadingSpinner />;
-  } else if (field.type.includes('range')) {
+  } else if (field.type.includes('range') || field.type === 'murmur3') {
     return (
       <>
         <EuiPopoverTitle>{panelHeader}</EuiPopoverTitle>
```

Comment:
The message specifically calls out "range type fields" which doesn't match the murmur3 case. Could you add a separate branch for this case with a new i18n phrase `Summary information is not available for murmur3 fields`?

Reply:
Just went ahead and pushed this change